### PR TITLE
Remove spec url form from /?doc swagger UI

### PR DESCRIPTION
**Pull request by ottomata**

### CODE REVIEW

File: lib/swagger-ui.js [hunk 1]
```diff
@@ -40,6 +40,8 @@ function processRequest(app, req, res) {
                 .replace(/Sorter: "alpha"/, 'Sorter: "alpha", validatorUrl: null, ' +
                     'highlightSizeThreshold: 10000')
                 .replace(/docExpansion: "none"/, 'docExpansion: "list"')
+                // Replace the spec url form. We will always use /?spec.
+                .replace(/\s*<form id='api_selector'>[\s\S]*<\/form>/m, '');
```
Comment (Pchelolo) [inline lib/swagger-ui.js]: The semicolon

**Review by Pchelolo — CHANGES_REQUESTED** [review]

**Comment by gwicke** [comment]
The spec is intended to be used for various automated tools, and a precondition for being able to consume the spec is the ability to discover the URL. This patch seems to make discovering the URL significantly harder by removing all mention of it from the UI, and I'm not sure that this is something we'd want.

**Comment by ottomata** [comment]
Hm, interesting.  It seems weird to use a form to let folks change the spec that a service's `/?doc` endpoint displays.  Would it make more sense to leave the form in place, but just make it uneditable?

**Comment by gwicke** [comment]
@ottomata: No objections from me against making it read-only.

**Comment by d00rman** [comment]
> Would it make more sense to leave the form in place, but just make it uneditable?

+1

**Pull request closed without merging**